molecule/default/converge.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Converge
+  hosts: all
+  roles:
+    - role: ansible-django
+      django_system_user: "django_example_app"
+      django_system_user_home: "/home/{{ django_system_user }}"
+      django_recreate_virtual_env: false
+      django_codebase_path: "{{ django_system_user_home }}/app"
+      django_versioned_path: "{{ django_codebase_path }}-versioned"
+      django_checkout_path: "{{ django_versioned_path }}/{{ ansible_date_time['epoch'] }}"
+      django_git_url: "https://github.com/moshthepitt/picha"
```

Comment:
I doubt if this will work well.  Its a really old version of Django there.  The reason why I chose it initially was because it was an example of a really really simple Django project that is in a public repo.  Now that the Kaznet django app is in a public repo, maybe we should switch to that as the django app being installed in this test.